src/cata_tiles.cpp

```diff
@@ -680,34 +680,41 @@ void cata_tiles::load_ascii_set( JsonObject &entry, int offset, int size )
 
 void cata_tiles::load_tilejson_from_file( JsonObject &config, int offset, int size )
 {
-    if( !config.has_member( "tiles" ) ) {
+    if ( !config.has_member( "tiles" ) ) {
```

Comment:
I... really don't know what to do about this. I'm already spending a good fraction as much time on using astyle as I spend writing code. I'd really really like to just astyle the whole project and make this a non-issue.